### PR TITLE
Add --create-sample-syncsets to hiveutil create-cluster

### PR DESCRIPTION
**Pull request by abutcher**

### CODE REVIEW

File: contrib/pkg/createcluster/create.go [hunk 1]
```diff
@@ -79,6 +79,7 @@ const (
 	deleteAfterAnnotation      = "hive.openshift.io/delete-after"
 	tryInstallOnceAnnotation   = "hive.openshift.io/try-install-once"
 	tryUninstallOnceAnnotation = "hive.openshift.io/try-uninstall-once"
+	hiveutilCreatedLabel       = "hive.openshift.io/hiveutil-created"
```
Comment (abutcher) [inline contrib/pkg/createcluster/create.go]: I need something to match when creating selectorsyncsets so added this label.

File: contrib/pkg/createcluster/create.go [hunk 2]
```diff
@@ -643,6 +653,76 @@ func (o *Options) configureImages(cd *hivev1.ClusterDeployment) (*hivev1.Cluster
...
+
+func sampleCM(name string) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: "default",
```
Comment (abutcher) [inline contrib/pkg/createcluster/create.go]: Everything lands in default namespace.

File: contrib/pkg/createcluster/create.go [hunk 3]
```diff
@@ -643,6 +653,76 @@ func (o *Options) configureImages(cd *hivev1.ClusterDeployment) (*hivev1.Cluster
 	return imageSet, nil
 }
 
+func (o *Options) generateSampleSyncSets() []runtime.Object {
+	syncsets := []runtime.Object{}
+	for i := range [10]int{} {
```
Comment (abutcher) [inline contrib/pkg/createcluster/create.go]: Randomly went with 10 of each.

File: contrib/pkg/createcluster/create.go (hunk 1 above)
Comment (dgoodwin) [reply contrib/pkg/createcluster/create.go]: Good idea.

**Comment by dgoodwin** [comment]
/lgtm

Done on master which was perfect for tomorrow, thanks.

**Pull request merged**